docs/getting-started/04-expose-microservice.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Expose the microservice
+type: Getting Started
+---
+
+Now that you deployed a standalone `orders-service` microservice, allow other resources to communicate with it. Make it available for other resources outside the cluster by exposing its Kubernetes Service through an APIRule custom resource (CR).
+
+## Reference
+
+This guide demonstrates how [API Gateway](/components/api-gateway) works in Kyma. It exposes your Service under secured or unsecured endpoints for external Services to communicate with it.
```

Comment:
```suggestion
This guide demonstrates how [API Gateway](/components/api-gateway) works in Kyma. It exposes your Service on secured or unsecured endpoints for external Services to communicate with it.
```
You expose something `on an endpoint` or `at an endpoint`. See also [this ngram](https://books.google.com/ngrams/graph?year_end=2019&year_start=1800&content=under+an+endpoint%2C+on+an+endpoint%2C+at+an+endpoint&smoothing=3&corpus=26&direct_url=t1%3B%2Con%20an%20endpoint%3B%2Cc0%3B.t1%3B%2Cat%20an%20endpoint%3B%2Cc0). 

 In these guides, you use `on`, so I went with this one for consistency.

Reply:
Thx, changed it to `on` everywhere.